src/Plugins/RFC.php

```diff
@@ -207,6 +191,87 @@ public function getRFC(Command $command)
         return $this->chatClient->postMessage($command->getRoom(), $message);
     }
 
+    public function getRFCVotes(Command $command)
+    {
+        $rfc = $command->getParameter(0);
+
+        if ($rfc === null) {
+            return $this->chatClient->postMessage($command->getRoom(), "Usage: `!!voting <id>`");
+        }
+
+        $uri = self::BASE_URI . '/' . urlencode($rfc);
+
+        /** @var HttpResponse $response */
+        $response = yield $this->httpClient->request($uri);
+
+        if ($response->getStatus() !== 200) {
+            return $this->chatClient->postMessage($command->getRoom(), "Nope, we can't have nice things.");
+        }
+
+        $messages = $this->prepareVotes($response->getBody(), $uri);
```

Comment:
Honestly I'm not a fan of this. I bundled repeated code together into `prepareVotes`. Maybe it could be renamed and refactored?